JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/Services/IAuthenticationService.cs

```diff
@@ -7,7 +9,8 @@ public interface IAuthenticationService
     {
         bool IsAuthenticated { get; set; }
         UserModel CurrentUser { get; set; }
+        HttpClient _httpClient { get; set; }
```

Comment:
httpClient is already an injected singleton with mvvmcross you never need to make it public